node_build/dependencies/get_libuv.sh

```diff
@@ -23,12 +23,12 @@ die() {
     exit 100;
 }
 
-git clone http://github.com/joyent/libuv.git || die 'clone libuv'
+rm -rf libuv
+git clone http://github.com/joyent/libuv.git -b v1.x || die 'clone libuv'
```

Comment:
@guworks Yea isn't this the patch you want? https://github.com/libuv/libuv/commit/d57c7e685f9ef2f00aa0cd5661194e8059e84d00